packages/toggle-crud-psr11-factories/src/FeatureFinderFactory.php

```diff
@@ -19,6 +19,13 @@ public function __invoke(ContainerInterface $container): FeatureFinder
     {
         /** @var ToggleConfig $config */
         $config = $container->get(ToggleConfig::class);
+        /** @var Connection $connection */
+        $connection = $container->get(Connection::class);
+
+        return self::create($config, $connection);
+    }
+
+    public static function create(ToggleConfig $config, ?Connection $connection): FeatureFinder {
```

Comment:
Be aware of PSR-12 coding standard:
```suggestion
    public static function create(ToggleConfig $config, ?Connection $connection): FeatureFinder 
    {
```

Also, you can run the composer cs-fix command to fix coding standards automatically.